.github/workflows/test.yml

```diff
@@ -127,3 +127,16 @@ jobs:
           bin/vl2png < examples/specs/bar.vl.json > bar-stdin.png
           bin/vl2svg < examples/specs/bar.vl.json > bar-stdin.svg
           bin/vl2pdf < examples/specs/bar.vl.json > bar-stdin.pdf
+
+  docker-check:
+      name: docker
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Dockerfile
+        run: docker build -t vega-lite -f .devcontainer/Dockerfile .
```

Comment:
There is a tool which "builds" the image as VSCode would do it (not only using teh Dockerfile, but the setup).
https://code.visualstudio.com/docs/remote/devcontainer-cli

I tried it, it works. 
It is even better for the purpose of "exercising" the devcontainer setup.

Reply:
Should I add it ?

Reply:
I added it as one more commit